src/components/Header/Header.jsx

```diff
@@ -1,17 +1,27 @@
-import React from 'react';
-import { Link } from 'react-router-dom';
-import styles from './Header.module.css';  
+import React, { useState } from "react";
+// import { Link } from 'react-router-dom';
+import icon_close from "../../assets/icon-close.svg";
+import logo from "../../assets/logo.svg";
+import styles from "./Header.module.css";
 
 const Header = () => {
+  const [mobileMenu, setMobileMenu] = useState(false);
+  const toggleMenu = () => {
+    mobileMenu ? setMobileMenu(false) :
+   setMobileMenu(true);
+  };
   return (
     <header className={styles.header}>
+      <img src={logo} alt="myteam logo" className={styles.logo} />
       <nav>
-        <ul>
-          <li><Link to="/">Home</Link></li>
-          <li><Link to="/about">About</Link></li>
-          <li><Link to="/contact">Contact Us</Link></li>
+        <ul className={ mobileMenu ? '' :  styles['hide-mobile-menu']}>
+          <img src={icon_close} alt="" className={styles.icon_close} onClick={toggleMenu} />
+         <li>home</li>
+          <li>about</li>
         </ul>
+          <button className={styles.btn}>contact us</button>
```

Comment:
Please wrap the button in a `<Link to="/contact">` to enable navigation. This will get it working as navigation, and that's enough for us on this project right now.

Ideally, for semantic HTML and best-practice accessibility, we would style the link without an actual `button`, but for us for now, we'll gloss over that to get the component working and looking right. If we have time later, we can look at our best option for this.